Docs/Tor.md

```diff
@@ -0,0 +1,219 @@
+## Connecting over Tor 
+ - [macOS](#Connecting-over-Tor-macOS)
+ - [Windows 10](#Connecting-over-Tor-Windows-10)
+ - [Debian 10](#Connecting-over-Tor-Linux-Debian-10)
+ 
+#### Optional Tor v3
+## Connecting over Tor macOS
+Run `brew --version` in a terminal, if you get a valid response you have brew installed already. If not, install brew:
+
+```cd /usr/local
+mkdir homebrew && curl -L https://github.com/Homebrew/brew/tarball/master | tar xz --strip 1 -C homebrew
+```
+### On the device running your node:
+- run `brew install tor` in a terminal
+- Once Tor is installed you will need to create a Hidden Service.
+- Now first locate your `torrc` file, this is Tor's configuration file. Open Finder and type `shift command h` to navigate to your home folder and  `shift command .` to show hidden files.
+-  If you've not been able to locate the torrc file, you might have to create the torrc file manually first. Do this by copying the torrc.sample -file: `cp /usr⁩/local⁩/etc⁩/tor⁩/torrc.sample /usr⁩/local⁩/etc⁩/tor⁩/torrc` and give the file it's right permission `chmod 700 /usr⁩/local⁩/etc⁩/tor⁩/torrc`
+- The torrc file should be located at `‎⁨/usr⁩/local⁩/etc⁩/tor⁩/torrc`, to edit it you can open terminal and run `sudo nano /usr⁩/local⁩/etc⁩/tor⁩/torrc`
+- Locate the section that looks like:
+
+```
+## Once you have configured a hidden service, you can look at the
+## contents of the file ".../hidden_service/hostname" for the address
+## to tell people.
+##
+## HiddenServicePort x y:z says to redirect requests on port x to the
+## address y:z.
+
+```
+
+- And below it add one Hidden Service for each port:
+
+```
+HiddenServiceDir /usr/local/var/lib/tor/fullynoded/main
+HiddenServiceVersion 3
+HiddenServicePort 8332 127.0.0.1:8332
+
+HiddenServiceDir /usr/local/var/lib/tor/fullynoded/test
+HiddenServiceVersion 3
+HiddenServicePort 18332 127.0.0.1:18332
+
+HiddenServiceDir /usr/local/var/lib/tor/fullynoded/regtest
+HiddenServiceVersion 3
+HiddenServicePort 18443 127.0.0.1:18443
+
+HiddenServiceDir /usr/local/var/lib/tor/fullynoded/lightning
```

Comment:
can probably keep this one in lightning.md to keep confusion to a minimum, lightning setup is a totally different animal